modules/registry/examples_test.go

```diff
@@ -77,26 +80,27 @@ func ExampleRun_withAuthentication() {
 				BuildArgs: map[string]*string{
 					"REGISTRY_HOST": &registryHost,
 				},
-				PrintBuildLog: true,
```

Comment:
Same here: do we want to skip printing the build logs?

Reply:
Again just made it difficult to debug, and not needed any more for error handling as that's been fixed.